test-file.txt

```diff
@@ -1,19 +1,16 @@
 1.
-2.
 3.
 4.
 5.
-6.
+6. Line 2 was deleted, so this is now line 5
+And here's a new line on what is now line 6
```

Comment:
Is the bug fixed? This comment should be on the line that says "And here's a new line on what is now line 6"